server/lib/src/be/dbvalue.rs

```diff
@@ -211,16 +212,17 @@ pub enum DbCred {
 
     // New Formats!
     #[serde(rename = "V2Pw")]
-    V2Password { password: DbPasswordV1, uuid: Uuid },
+    V2Password { password: DbPasswordV1, uuid: Uuid, timestamp: OffsetDateTime },
```

Comment:
This will fail  to deserialise existing credentials as this fields doesn't exist. You probably want to annotate this with `#[serde(default = unix_epoch)]` and then make a fn for unix_epoch() -> OffsetDateTime so that when no timestamp is there, we default to epoch.